### PR TITLE
DSRC-90: PersonPage info

### PR DESCRIPTION
**Pull request by jamesbiggs**
Ticket URL: [DSRC-90]

## About these changes

Adds `research_summary` to the `PersonPage`, and then adds some logic for the `role_tags` using the new `is_X` properties.

Also adds a `first_` and `last_name` field to the promote panels, which are currently defaulting to `"first"` and `"name"` as we needed to have a default in there as they are non-nullable.

If the page (person) has authored some articles, they are assigned the `is_author` property. If the person has got some text in the `research_summary` field, then they are a researcher.

In the CMS:
image
image

In the API:
image
image

## How to check these changes

Where possible, provide guidance to help your reviewer

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [x] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

[DSRC-90]: https://national-archives.atlassian.net/browse/DSRC-90?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: etna/people/models.py [hunk 1]
```diff
@@ -57,24 +64,41 @@ class PersonPage(BasePage):
...
         null=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
+    role = models.CharField(blank=True, null=True, max_length=100)
+    summary = RichTextField(
+        blank=True, null=True, features=settings.RESTRICTED_RICH_TEXT_FEATURES
+    )
+
+    research_summary = StreamField(ResearchSummaryStreamBlock, blank=True, null=True)
```
Comment (ahosgood) [inline etna/people/models.py:79]: Is this the structure the content team wanted?

I was under the impression it was something more formulaic like:

- `research_activity` - optional single richtext
- `select_publications` - optional list of richtext fields
- `specialism` - optional list of text fields

Comment (ahosgood) [reply etna/people/models.py:79]: That's cool then. Thank you for checking 👍🏼

**Review by ahosgood — APPROVED** [review]

**Comment by ahosgood** [comment]
@jamesbiggs - I've just thought... Is this meant to contain contact info as well or did we say we were going to omit that?

**Comment by jamesbiggs** [comment]
@ahosgood We're going to omit that, a few stakeholders said they didn't want it to be there

**Comment by ahosgood** [comment]
@jamesbiggs Another thing (sorry I'm so bad at this!) - could we order the people in the people index page by last name?

**Comment by jamesbiggs** [comment]
Done! 😃 @ahosgood

**Comment by ahosgood** [comment]
Super-approve ✅

File: etna/people/models.py [hunk 2]
```diff
@@ -57,24 +64,41 @@ class PersonPage(BasePage):
...
-    )
     image = models.ForeignKey(
         get_image_model_string(),
         blank=True,
         null=True,
         on_delete=models.SET_NULL,
         related_name="+",
     )
+    role = models.CharField(blank=True, null=True, max_length=100)
+    summary = RichTextField(
```
Comment (ahosgood) [inline etna/people/models.py:75]: Should we make this required?

Comment (jamesbiggs) [reply etna/people/models.py:75]: They may not want an image (I think Lizzy said she'd like a default/silhouette type thing) but I will make role and summary required 🙂

**Pull request merged**